src/xprof.app.src

```diff
@@ -1,13 +1,17 @@
 {application, xprof,
  [{description, "Visual BEAM profiler"}
- ,{vsn, "1.1.0"}
+ ,{vsn, "1.2.0"}
  ,{registered, []}
  ,{mod, {'xprof_app', []}}
  ,{applications,
    [kernel
    ,stdlib
    ,cowboy
    ,lager
+   ,jsone
+   ,hdr_histogram
+   ,edown
```

Comment:
edown and fqc are just doc and test dependencies of hdr_histogram so probably we don't need them (especially not in a release)
or are they needed to be listed for the hex package?